README.md

```diff
@@ -3,3 +3,48 @@
 [![Build Status](https://travis-ci.org/coreos/jetcd.svg?branch=master)](https://travis-ci.org/coreos/jetcd)
 
 Java client for etcd v3.
+
+## Get Started
+
+### Download
```

Comment:
yes, we can remove it now, once we have upload it to central maven, we can add this back.